Managers/FrameMan.cpp

```diff
@@ -1242,9 +1222,10 @@ namespace RTE {
 					BITMAP *targetBitmap = g_PostProcessMan.GetTempEffectBitmap(effectBitmap);
 					clear_to_color(targetBitmap, 0);
 
-					 fixed fAngle = fixmul(postEffect.m_Angle, radtofix_r);
+					Matrix newAngle;
```

Comment:
Pretty sure you can oneline this definition to `Matrix newAngle(postEffect.m_Angle);`